pyvista/conftest.py

```diff
@@ -10,3 +10,11 @@ def autoclose_plotters():
     """Close all plotters."""
     yield
     pyvista.close_all()
+
+
+@pytest.fixture(autouse=True)
+def reset_gloal_theme():
+    """Reset global_theme."""
+    # this stops any doctest-module tests from overriding the global theme and
+    # creating test side effects
```

Comment:
Since this is purely for teardown, just like `autoclose_plotters()`, shouldn't this also be a yield fixture too?
```suggestion
    # creating test side effects
    yield
```